### DISCUSSION TITLE
File Format and CTD module for Regulatory Submission of PBPK report to US-FDA

### DISCUSSION THREAD

**Discussion by arketkar** [Open-Systems-Pharmacology/Forum#1063] [Questions & Problems] [answered]
I would like to submit a PBPK analysis report in support of a request to waive a clinical PK study for a drug-product to be filed in the US. Can someone please guide me on the following?
(1) Which file format should be used to upload the PK-Sim files to US-FDA's server? 
(2) Is '.pksim5' format acceptable to US-FDA? It is currently not listed in the guidance document issued by FDA- 'Specifications for File Format Types Using eCTD Specifications' (https://www.fda.gov/media/85816/download)
(3) In which CTD Module the PBPK analysis report and the supporting electronic files (including .pksim5 files) are to be uploaded?
(4) Any other suggestions or expert advice or tips?

I would appreciate if someone having experience in Regulatory Submission of PK-Sim based reports could share her/his experience.

Thanks.
Anant

**Comment by prvmalik** [comment]
It's a good question. I will do this myself in January and let you know how it goes. No reason .pksim5 would not be acceptable if the SimCyp and Gastroplus file formats are listed in that directory. If there are formatting issues or size limits in the directory, my plan is to .zip it.

**Comment by arketkar** [comment]
@prvmalik 
Thanks for your response. Right, SimCyp and Gastroplus file formats are listed. However surprisingly there is neither a mention of other platforms nor any solution or note provided for other platforms.
What I got to know from Regulatory team is that they were not able to upload .pksim5 and getting an error message. I am not sure whether a .zip file of .pksim5 will be accepted. Will try and let you know. Right now we are stuck....
Regards.

**Comment by msevestre** [comment]
We need to do what's required to get PKSim and MoBi on that list. I am fairly confident that zip files can be uploaded in the meantime.

**Comment by arketkar** [comment]
@msevestre 
Sounds like a great idea! This would be a great help.
Regarding zip files, I will keep all posted on the acceptance of the application and the way of upload. Appreciate your inputs.
Regards.

**Comment by arketkar** [comment, marked as answer]
@prvmalik and @msevestre 

I got a confirmation from Regulatory team that the .pksim5 files have cleared the FDA submission gateway! So its a good news for now.
That means you need not even zip the files 👍 
Thank you both for your inputs. Appreciate that.

Regards.

**Reply by msevestre** [reply]
Awesome. We need to make sure that the mobi files are also accepted. Of course, the files should be in this list. Thanks for your feedback @arketkar

**Reply by StephanSchaller** [reply]
@arketkar, can you share your experience with the submission of PK-Sim / OSP Suite files to FDA? 
Please also see our [call for regulatory examples](https://github.com/Open-Systems-Pharmacology/Forum/discussions/1135)!

**Reply by prvmalik** [reply]
@arketkar what eCTD module did you puts reports and files in?

**Reply by arketkar** [reply]
Hi @prvmalik 

The reports were uploaded in Module 5.3.1.3.

For that we took a reference from Pg. 48 of 57 of the file from the following link.
https://www.accessdata.fda.gov/drugsatfda_docs/nda/2017/209203Orig1s000ChemR.pdf

Hope this helps you and others in the forum!
